src/extension/tests/Test_ActionHandler.py

```diff
@@ -214,8 +214,14 @@ def test_telemetry_not_available(self):
         with self.assertRaises(SystemExit) as sys_exit:
             self.action_handler.enable()
 
-        # event folder is set within HandlerEnvironment but the directory path is invalid. i.e. Telemetry setup not valid
-        self.action_handler.ext_env_handler.events_folder = "testfolder"
+        self.action_handler.ext_env_handler.events_folder = events_folder_path_backup
```

Comment:
remove line 217. Earlier check was for the case when event folder path is set in HandlerEnvironment but does not contain a valid path. Which we are fixing by creating the events folder. So the test 'test_telemetry_not_available' will only validate when events folder path is not given.

Rename the new test below from 'test_telemetry_available_events_folder_not_exists' to 'test_telemetry_available_but_events_folder_not_exists'

Reply:
Gotcha, fixed and updated